packages/terra-clinical-application-site/src/Index.jsx

```diff
@@ -0,0 +1,10 @@
+import 'babel-polyfill';
```

Comment:
In `terra-site` and `terra-clincial-site`, the `babel-polyfill` import was moved out of src and into it's own webpack chunk. This might be something we want to do here as well. [Core PR](https://github.com/cerner/terra-core/pull/406/files) and [Clinical PR](https://github.com/cerner/terra-clinical/pull/93).